includes/Integrations/New_Relic.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Class New_Relic
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Integrations;
+
+use Google\Web_Stories\Infrastructure\Conditional;
+use Google\Web_Stories\Service_Base;
+
+/**
+ * New Relic integration class.
+ *
+ * @since 1.10.0
+ */
+class New_Relic extends Service_Base implements Conditional {
+	/**
+	 * Runs on instantiation.
+	 *
+	 * @since 1.10.0
+	 *
+	 * @return void
+	 */
+	public function register() {
+		// Run at the same time as the output buffering.
+		$priority = defined( 'PHP_INT_MIN' ) ? PHP_INT_MIN : ~PHP_INT_MAX; // phpcs:ignore PHPCompatibility.Constants.NewConstants.php_int_minFound
+		add_action( 'template_redirect', [ $this, 'disable_autorum' ], $priority );
+	}
```

Comment:
```suggestion
	public function register() {
		newrelic_disable_autorum();
	}
	
	 /**
	 * Get the action to use for registering the service.
	 *
	 * @since 1.6.0
	 *
	 * @return string Registration action to use.
	 */
	public static function get_registration_action(): string {
		return 'template_redirect';
	}
	
	/**
	 * Get the action priority to use for registering the service.
	 *
	 * @since 1.6.0
	 *
	 * @return int Registration action priority to use.
	 */
	public static function get_registration_action_priority(): int {
	    $priority = defined( 'PHP_INT_MIN' ) ? PHP_INT_MIN : ~PHP_INT_MAX; // phpcs:ignore PHPCompatibility.Constants.NewConstants.php_int_minFound
		return $priority;
	}
```

We could do this.